node/child_process.ts

```diff
@@ -140,6 +143,49 @@ export function spawn(
   return new ChildProcess(command, args, options);
 }
 
+function validateTimeout(timeout?: number) {
+  if (timeout !== undefined && !(Number.isInteger(timeout) && timeout >= 0)) {
+    throw new ERR_OUT_OF_RANGE("timeout", "an unsigned integer", timeout);
+  }
+}
+
+function validateMaxBuffer(maxBuffer?: number) {
+  if (
+    maxBuffer !== undefined &&
```

Comment:
To match the Node implementation:

```suggestion
    maxBuffer != null &&
```